### PR TITLE
Update subscriptions list test coverage

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Update subscriptions list test coverage

Handle plan not found case by using
product_id

[stage-14]

## Motivation and Context
Fix test coverage and edge case.

## How Has This Been Tested?
Tested locally. tests pass.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@santiagonoguez please review. I'm not sure why the tests show as Pending, in CCI the build is successful. Thanks.

File: web/scripts/billing/controllers/ctr-billing.js [hunk 1]
```diff
@@ -78,7 +87,7 @@ angular.module('risevision.apps.billing.controllers')
         var name = plan ? plan.name : subscription.plan_id;
         
         // Show `1` plan_quantity for Per Display subscriptions
-        if (_isVolumePlan(plan) && subscription.plan_quantity > 0) {
+        if ((plan && _isVolumePlan(plan) && subscription.plan_quantity > 0) || subscription.plan_quantity > 1) {
```
Comment (santiagonoguez) [inline web/scripts/billing/controllers/ctr-billing.js:90]: Then it seems that the original prefix var declaration above could be simplified to:
var prefix = '';

**Review by santiagonoguez — APPROVED** [review]
@alex-deaconu tests and changes look good to me, just a minor suggestion, thanks !

File: web/scripts/billing/controllers/ctr-billing.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/billing/controllers/ctr-billing.js:90]: @santiagonoguez oh, I didn't notice that prefix was already being initialized like that. I will remove this change, good catch. Thanks.

**Comment by alex-deaconu** [comment]
@santiagonoguez I've removed the redundant check you mentioned, and improved the trial filtering by using an actual filter. Please review the last commit. Thanks.
